stable/redis-ha/values.yaml

```diff
@@ -33,11 +33,21 @@ servers:
 
 rbac:
   # Specifies whether RBAC resources should be created
-  create: true
+  create: false
 
 serviceAccount:
   # Specifies whether a ServiceAccount should be created
-  create: true
+  create: false
   # The name of the ServiceAccount to use.
   # If not set and create is true, a name is generated using the fullname template
   name:
+
+## Configures redis with AUTH (requirepass & masterauth conf params)
+auth: false
+
+## Redis password
+## Defaults to a random 10-character alphanumeric string if not set and auth is true
+## ref: https://github.com/bitnami/bitnami-docker-redis#setting-the-server-password-on-first-run
```

Comment:
You should probably remove this ref since this is a different container

Reply:
done